ocaml/testsuite/tests/typing-unique/overwriting_underscore.ml

```diff
@@ -11,7 +11,7 @@ let underscore () = _
 Line 1, characters 20-21:
 1 | let underscore () = _
                         ^
-Error: Syntax error: wildcard "_" not expected.
+Error: Syntax error: "wildcard "_"" not expected.
```

Comment:
Hrm. That doesn't look right.

Reply:
This is from the `main` branch. I vaguely remember reading a slack message about this change?

Reply:
Yes, but I don't think there should be nested quotes.